server/glasses.js

```diff
@@ -0,0 +1,111 @@
+'use strict'
+
+const db = require('APP/db');
+const Glasses = db.model('glasses');
+const router = require('express').Router();
+
+
+
+//Create:
+//add glasses to database
+router.post('/admin/new-glasses', (req, res, next) => {
+  Glasses.create(req.body)
+    .then(createdGlasses => {
```

Comment:
.then and .catch calls should be at the same indentation as the promise call ("Glasses.create(req.body)" in this case)